bin/codebasin

```diff
@@ -144,11 +145,11 @@ def main():
             "--dump will be removed in a future release.",
             DeprecationWarning,
         )
-    if len(args.reports) > 1:
-        warnings.warn(
-            "Passing more than one value to --report (-R) is deprecated.",
-            DeprecationWarning,
-        )
+
+    # If no specific report was specified, generate all reports.
```

Comment:
I'm fine either way, but do you want to keep a warning message here to say "this is the default behavior if you don't specify any reports"?

Reply:
I thought about this, but decided that a warning would actually be more trouble than it's worth.  If we want to remove the option to write `--report all`, we either need to release a 1.3.0 (with a deprecation warning) before 2.0.0, or wait until 3.0.0 to remove this option.

But you're right that we should document what the default behavior is.  So, I added a note to the help string in 86b57a5d27d0a6cc26cc1ff668652eca2f04fb4c.